mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Placebo.MixProject do
   def project do
     [
       app: :placebo,
-      version: "2.0.0-rc.2",
+      version: "2.1.0",
```

Comment:
Change this to `2.0.0`. This is a bug fix at best, so `2.0.1` would be more appropriate than `2.1.0`. But since the current version is an RC, we can include the fix in the release of `2.0.0`.